storage/rocksdb/rdb_threads.cc

```diff
@@ -61,17 +64,6 @@ int Rdb_thread::create_thread(const std::string &thread_name
   int err = mysql_thread_create(background_psi_thread_key, &m_handle, nullptr,
                                 thread_func, this);
 
-  if (!err) {
```

Comment:
Well, if you want it, I'll just go ahead and add a BP for PS to expose the call, implement it and change this up to use it. I don't find it particularly useful, I've lived without it or years so far and managed to debug things :-\